### ISSUE TITLE
No code in the repo?

### ISSUE THREAD

**Issue by HammyHavoc** [simachri/taskwarrior-ms-todo#1]
Hi! This sounds useful, but there's no code in the repo. Did you not go ahead with this in the end?

**Comment by simachri** [comment]
No, I have not found sufficient time to proceed with this project yet. I have created the repo at the end of my last vacation but now my regular work got in my way. I might find time to work on this by the end of this year. 

However, thank you for your interest!
